### ISSUE TITLE
URL escape error if gotify is on domain subpath

### ISSUE THREAD

**Issue by tripleFA** [Ondolin/gotify-matrix-bot#12]
Hi,
Since you switched to websocket the bot can't connect to webserver. Gotify is on a subpath : https://domain.tld/gotify. 

I got this error : 

```
Error while trying to connect to the webserver:parse "wss://domain.tld%2Fgotify/stream?token=xxxxxx_xxxxxxxx": invalid URL escape "%2F"
```

**Comment by Ondolin** [comment]
I'll try to fix this later this day.

**Comment by Ondolin** [comment]
Should be solved since version 2.0.2. Please reopen if it is still present.
